ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -384,6 +402,12 @@ def _router_bgp_vlan_aware_bundles(self) -> list | None:
             return None
 
         if not self._evpn_vlan_aware_bundles:
+            first_evpn_vlan_bundle = self._get_first_evpn_vlan_bundle
+            if first_evpn_vlan_bundle is not None:
+                raise AristaAvdMissingVariableError(
+                    "Option 'evpn_vlan_aware_bundles' must be set to 'true' to be able to use 'evpn_vlan_bundle' option in l2vlans. "
+                    f"First occurence of 'evpn_vlan_bundle' seen for l2vlan {first_evpn_vlan_bundle['id']} in Tenant '{first_evpn_vlan_bundle['tenant']}'."
+                )
             return None
```

Comment:
@carlbuchmann This check will affect all AVD runs, since it happens even if the feature is disabled. It loops through all l2vlans and checks for a key, just to create the error. I think we need to rethink this, and maybe accept that we just ignore the key for now, and later we can have the schema check these kind of things.

Reply:
agreed